Eastern-Research-Group/expert-query
# Feature/getting ready for scan

## Related Issues:
* None

## Main Changes:
* Changed the status code for the `NoParametersException` to `200` for security scanning purposes.
* Changed the status code for the `The column ... does not exist on the selected profile` message to `400 Bad Request`, instead of `404 Not Found`. 
* Updated the etl to build a materialized view that precomputes the counts for each profile.
* Updated the count endpoints to use the new count materialized view when no filters are provided, should provide better performance.
* Fixed issues with server crashing when postgres errors occur.
* Removed the `GET` version of the `{profile}\values` endpoint.
* Updated the private open api document.

## Steps To Test:
1. Run the etl against your local database
2. Start the app
3. Verify everything works
4. Verify the updates in the open api file looks good

## TODO:
- [ ] Rerun the etl on the dev site after this is merged.
